native_ref/pom.xml

```diff
@@ -27,6 +28,37 @@
           <!-- cross compiles are built separately on a Linux box... -->
 	</modules>
       </profile>
+      <profile>
```

Comment:
this is not how the release workflow works. It needs to be performed over multiple OS, which is why making a release is so time consuming.